src/content/en/fundamentals/native-hardware/capturing-images/index.md

```diff
@@ -194,11 +191,13 @@ If the user has not previously granted your site access to the camera then
 the instant that you call `getUserMedia` the browser will prompt the user to
 grant your site permission to the camera. 
 
-User's hate getting prompted for access to powerful devices on their machine and
-they will frequently block the request, or they will ignore it if they don't 
-understand the context of which the prompt has been created. It is best practice
-to only ask to access the camera when first needed. Once the user has
-granted access they won't be asked again, however, if they reject access, 
-you can't get access again to ask the user for permission.
+Users hate getting prompted for access to powerful devices on their machine 
+and they will frequently block the request, or they will ignore it if they don't 
+understand the context for which the prompt has been created. It is best 
+practice to only ask to access the camera when first needed. Once the user has
+granted access they won't be asked again. However, if the user rejects access, 
+you can't get access again, unless they manually change camera permission 
+settings.
 
-Warning: Asking for access to the camera on page load will result in most of your users rejecting access to it.
+Warning: Asking for access to the camera on page load will result in most of 
```

Comment:
anecdotally: this kind of works if you explain why you want access. We did an A/B experiment on appear.in a while back asking for GUM permissions only after a button click and that did not significantly lower the permission denied errors.